docs/DeveloperGuide.md

```diff
@@ -373,96 +373,249 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+**MSS**
 
-    * Use case ends.
+1. User requests to **list persons (UC1)**.
+2. ContaX displays a list of persons.
+3. User requests to delete a person.
+4. User enters details to delete person.
+5. ContaX deletes the person.
+6. ContaX displays a message indicating that the person was successfully deleted.
+    Use case ends.
 
-* 1c. The person already exists
+**Extensions**
 
-    * 1c1. ContaX shows an error message indicating that the person already exists.
+* 1a. User filters person by attribute.
+  * 1a1. User **finds person (UC5)**.
+  * Use case resumes from step 2.<br>&nbsp;
 
-    * Use case ends.
+* 1b. User filters persons by tag keyword.
+  * 1b1. User **finds person by tag (UC10)**.
+  * Use case resumes from step 2.<br>&nbsp;
 
-**Use case: Delete a person**
+* 2a. ContaX has no persons to list.
+  * Use case ends.<br>&nbsp;
 
-**MSS**
+* 4a. ContaX detects that a required person attribute was not supplied.
+    * 4a1. ContaX shows an error message indicating that there is a missing required attribute.
+    * Use case ends.<br>&nbsp;
+
+* 5a. ContaX cannot find the requested person to delete.
+  * 5a1. ContaX shows an error message indicating that no such person exists.
+  * Use case ends.<br>&nbsp;
 
-1.  User requests to list persons
-2.  ContaX shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  ContaX deletes the person
+**UC4: Edit Person**
+
+**MSS**
 
+1. User requests to **list persons (UC1)**.
+2. ContaX shows a list of persons.
+3. User requests to edit person.
+4. User enters details to modify a specific person.
+5. ContaX updates the specified person.
+6. ContaX displays a message indicating that the person was successfully edited.
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty
+* 1a. User filters person by attribute.
+    * 1a1. User **finds person (UC5)**.
+    * Use case resumes from step 2.<br>&nbsp;
 
-  Use case ends.
+* 1b. User filters persons by tag keyword.
+    * 1b1. User **finds person by tag (UC10)**.
+    * Use case resumes from step 2.<br>&nbsp;
 
-* 3a. The command given is in an invalid format
+* 2a. ContaX has no persons to list.
+  * Use case ends.<br>&nbsp;
 
-    * 3a1. ContaX shows an error message indicating that the given command is of invalid format.
+* 4a. ContaX detects that a required person attribute was not supplied.
+    * 4a1. ContaX shows an error message indicating that there is a missing required attribute.
+    * Use case ends.<br>&nbsp;
+     
+* 4b. ContaX detects that a supplied attribute has an invalid value
+    * 4b1. ContaX shows an error message indicating that the supplied parameter is invalid.
+    * 4b2. ContaX shows the expected allowed values.
+    * Use case ends.<br>&nbsp;
 
-    * Use case resumes at step 2.
+* 5a. ContaX cannot find the requested person to delete.
+    * 5a1. ContaX shows an error message indicating that no such person exists.
+    * Use case ends.<br>&nbsp;
```

Comment:
If you want to say requested person to **edit** is not found, then you should place it at step 3.

Reply:
Fixed in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/263/commits/7f066dcdd276c2f95aac11a8da54e000f9667dba